src/traces/table/attributes.js

```diff
@@ -0,0 +1,278 @@
+                'https://github.com/d3/d3-format/blob/master/README.md#locale_format'
+            ].join(' ')
+        },
+
+        prefix: {
+            valType: 'string',
+            arrayOk: true,
+            dflt: null,
+            role: 'style',
+            description: 'Prefix for cell values.'
+        },
+
+        suffix: {
+            valType: 'string',
+            arrayOk: true,
+            dflt: null,
+            role: 'style',
+            description: 'Suffix for cell values.'
+        },
+
+        height: {
+            valType: 'number',
+            dflt: 20,
+            role: 'style',
+            description: 'The height of cells.'
+        },
+
+        align: extendFlat({}, annAttrs.align, {arrayOk: true}),
+        valign: extendFlat({}, annAttrs.valign, {arrayOk: true}),
+
+        line: {
+            width: {
+                valType: 'number',
+                arrayOk: true,
+                role: 'style'
+            },
+            color: {
+                valType: 'color',
+                arrayOk: true,
+                role: 'style'
+            }
+        },
+
+        fill: {
+            color: {
+                valType: 'color',
+                arrayOk: true,
+                role: 'style',
+                description: [
+                    'Sets the cell fill color. It accepts either a specific color',
+                    ' or an array of colors.'
+                ].join('')
+            }
+        },
+
+        font: {
+            family: {
+                valType: 'string',
+                arrayOk: true,
+                role: 'style',
+                noBlank: true,
+                strict: true,
+                description: [
+                    'HTML font family - the typeface that will be applied by the web browser.',
+                    'The web browser will only be able to apply a font if it is available on the system',
+                    'which it operates. Provide multiple font families, separated by commas, to indicate',
+                    'the preference in which to apply fonts if they aren\'t available on the system.',
+                    'The plotly service (at https://plot.ly or on-premise) generates images on a server,',
+                    'where only a select number of',
+                    'fonts are installed and supported.',
+                    'These include *Arial*, *Balto*, *Courier New*, *Droid Sans*,, *Droid Serif*,',
+                    '*Droid Sans Mono*, *Gravitas One*, *Old Standard TT*, *Open Sans*, *Overpass*,',
+                    '*PT Sans Narrow*, *Raleway*, *Times New Roman*.'
+                ].join(' ')
+            },
+            size: {
+                valType: 'number',
+                arrayOk: true,
+                role: 'style'
+            },
+            color: {
+                valType: 'color',
+                arrayOk: true,
+                role: 'style'
+            }
+        }
+    }
+};
```

Comment:
You'll need to add `editType` declarations to these attributes.

See https://github.com/plotly/plotly.js/pull/1999